gms/impl/src/main/java/com/linkedin/metadata/resources/dataset/Datasets.java

```diff
@@ -150,6 +150,7 @@ protected DatasetSnapshot toSnapshot(@Nonnull Dataset dataset, @Nonnull DatasetU
     if (dataset.hasDeprecation()) {
       aspects.add(ModelUtils.newAspectUnion(DatasetAspect.class, dataset.getDeprecation()));
     }
+
```

Comment:
I think this is unrelated to the PR, so better to avoid unrelated whitespace changes. Can fix in a follow up commit.